magicbell-rails.gemspec

```
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name        = 'magicbell-rails'
-  spec.version     = '0.2.0'
+  spec.version     = '0.2.1'
```

you adding new features so it would be a minor update and not a bug fix update

```suggestion
  spec.version     = '0.3.0'
```